### PR TITLE
Develop media plyr alex

### PR DESCRIPTION
**Pull request by atyrrell1**

### CODE REVIEW

**Review by szuk1 — APPROVED** [review]

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
```
Comment (rizwan146) [inline app/src/main/res/layout/activity_main.xml:4]: What is this used for?

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -21,12 +22,14 @@
     <fragment
         android:id="@+id/nav_host_fragment"
         android:name="androidx.navigation.fragment.NavHostFragment"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
+        android:layout_width="406dp"
```
Comment (rizwan146) [inline app/src/main/res/layout/activity_main.xml]: remove these fixed values

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -21,12 +22,14 @@
     <fragment
         android:id="@+id/nav_host_fragment"
         android:name="androidx.navigation.fragment.NavHostFragment"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
+        android:layout_width="406dp"
+        android:layout_height="635dp"
```
Comment (rizwan146) [inline app/src/main/res/layout/activity_main.xml]: remove these fixed values

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -21,12 +22,14 @@
     <fragment
         android:id="@+id/nav_host_fragment"
         android:name="androidx.navigation.fragment.NavHostFragment"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
+        android:layout_width="406dp"
+        android:layout_height="635dp"
         app:defaultNavHost="true"
         app:layout_constraintBottom_toTopOf="@id/nav_view"
+        app:layout_constraintEnd_toEndOf="parent"
```
Comment (rizwan146) [inline app/src/main/res/layout/activity_main.xml]: remove these fixed values

File: app/src/main/res/layout/activity_main.xml [hunk 5]
```diff
@@ -21,12 +22,14 @@
...
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
+        android:layout_width="406dp"
+        android:layout_height="635dp"
         app:defaultNavHost="true"
         app:layout_constraintBottom_toTopOf="@id/nav_view"
+        app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintLeft_toLeftOf="parent"
         app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
```
Comment (rizwan146) [inline app/src/main/res/layout/activity_main.xml]: remove these fixed values

File: app/src/main/res/layout/activity_main.xml (hunk 2 above)
Comment (rizwan146) [reply app/src/main/res/layout/activity_main.xml]: Actually, I don't think you can remove this. Change it to 0dp to indicate that the size is being changed somewhere else.

File: app/src/main/res/layout/activity_main.xml (hunk 3 above)
Comment (rizwan146) [reply app/src/main/res/layout/activity_main.xml]: Actually, I don't think you can remove this. Change it to 0dp to indicate that the size is being changed somewhere else.

File: app/src/main/res/layout/fragment_media_player.xml [hunk 6]
```diff
@@ -1,19 +1,134 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
```
Comment (rizwan146) [inline app/src/main/res/layout/fragment_media_player.xml]: What is this used for?

File: app/src/main/res/layout/fragment_media_player.xml [hunk 7]
```diff
@@ -1,19 +1,134 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
+    <ImageView
```
Comment (rizwan146) [inline app/src/main/res/layout/fragment_media_player.xml]: Move all these children into a vertical Linear Layout and set the layout weight accordingly. Then test it on multiple phones with varying screen sizes. This current layout does not fit on smaller phones.

**Review by rizwan146 — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_main.xml (hunk 2 above)
Comment (rizwan146) [reply app/src/main/res/layout/activity_main.xml]: Even better use "match_parent" again

File: app/src/main/res/layout/activity_main.xml (hunk 3 above)
Comment (rizwan146) [reply app/src/main/res/layout/activity_main.xml]: Even better use "match_parent" again

File: app/src/main/res/layout/activity_main.xml (hunk 4 above)
Comment (rizwan146) [reply app/src/main/res/layout/activity_main.xml]: end and start layout constraints are unnecessary when we have left and right layout constraints already

Comment (atyrrell1) [reply app/src/main/res/layout/activity_main.xml]: Sorry, but how is this a fixed value?

File: app/src/main/res/layout/activity_main.xml (hunk 2 above)
Comment (atyrrell1) [reply app/src/main/res/layout/activity_main.xml]: Using match_parent might be an issue when we retrieve the album art. Might stretch the image

Comment (atyrrell1) [reply app/src/main/res/layout/activity_main.xml]: match_parent works fine for the width. It doesn't work well for the height

File: app/src/main/res/layout/activity_main.xml (hunk 3 above)
Comment (atyrrell1) [reply app/src/main/res/layout/activity_main.xml]: match_parent for the height leaves a lot of white space at the top.

Comment (rizwan146) [reply app/src/main/res/layout/activity_main.xml]: Ahh, I see. I believe this is caused by `android:paddingTop="?attr/actionBarSize"` on line 7. Try removing that line and keep "match_parent" for height.

File: app/src/main/res/layout/activity_main.xml (hunk 4 above)
Comment (rizwan146) [reply app/src/main/res/layout/activity_main.xml]: My bad, was copying and pasting ;P
See my comment after it

**Pull request merged**